code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -521,3 +521,45 @@
 	if(index == 1 && target.address == text && target.active)
 		if(target.wearer && target.wearer.stat != DEAD)
 			return target
+
+/datum/ritual/cruciform/priest/buy_disk
+	name = "Order armaments"
+	phrase = "I need suggestions for a phrase"
```

Comment:
Need suggestions for the litany phrase

Reply:
"et qui non habet, vendat tunicam suam et emat gladium." It's from Luke 22:36, from the Vulgate Latin bible